be/src/vec/columns/column_dictionary.h

```diff
@@ -304,10 +310,14 @@ class ColumnDictionary final : public COWHelper<IColumn, ColumnDictionary<T>> {
             return -1;
         }
 
+        T get_null_code() {
+            return _dict_data.size() - 1; // The last dict value is null value
+        }
```

Comment:
For range predicate, it seems related to null order, I am not sure how doris handle null order. e.g. where col >= "xxx" includes null.